### PR TITLE
Docs/cors

### PR DESCRIPTION
**Pull request by jwoertink**
Added in a small doc about adding CORS headers to the response in API.

### CODE REVIEW

File: source/guides/writing-json-apis.html.md.erb [hunk 1]
```diff
@@ -185,3 +185,23 @@ head Status::Created
...
+
+  def set_cors_headers
+    response.headers["Access-Control-Allow-Origin"] = "*"
+    response.headers["Access-Control-Allow-Headers"] = "DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range"
+    continue
+  end
+end
+```
+
+> Note: the before action method must return `continue`, or a `LuckyWeb::Response`. See Actions and Routing for more info.
```
Comment (paulcsmith) [inline source/guides/writing-json-apis.html.md.erb]: Could you link to guides/actions-and-routing/#run-code-before-or-after-actions-with-pipes in the "See [Actions and Routing]"?

**Review by paulcsmith — COMMENTED** [review]
I think this is perfect. The link to Mozilla is great and it shows just enough to get people going with whatever they need 👍

File: source/guides/writing-json-apis.html.md.erb (hunk 1 above)
Comment (jwoertink) [reply source/guides/writing-json-apis.html.md.erb]: oops! I meant to add that. Got caught up switching between branches lol

File: source/guides/writing-json-apis.html.md.erb [hunk 2]
```diff
@@ -185,3 +185,23 @@ head Status::Created
...
+
+  def set_cors_headers
+    response.headers["Access-Control-Allow-Origin"] = "*"
+    response.headers["Access-Control-Allow-Headers"] = "DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range"
+    continue
+  end
+end
+```
+
+> Note: the before action method must return `continue`, or a `LuckyWeb::Response`. See [Actions and Routing](actions-and-routing/#run-code-before-or-after-actions-with-pipes) for more info.
```
Comment (paulcsmith) [inline source/guides/writing-json-apis.html.md.erb]: I think the link needs to start with `/`. Right now it is appending to the current path

Comment (jwoertink) [reply source/guides/writing-json-apis.html.md.erb]: Strange. I did that first and it was sending me to `localhost:4567/actions-and...`. Maybe I'll do `/guides/actions-and` instead

**Comment by paulcsmith** [comment]
That did it!

**Pull request merged**